pkg/common/constants.go

```diff
@@ -32,8 +32,10 @@ const (
 	MonitoringNamespace = "everest-monitoring"
 	// PerconaEverestDeploymentName stores the name of everest API Server deployment.
 	PerconaEverestDeploymentName = "percona-everest"
```

Comment:
I know that I should have commented this in the PR of the helm chart but I'm only paying attention to it now that I commented about the operator name below. Can we change this name without breaking stuff? :sweat_smile:

Reply:
At least within Everest, I don't think it breaks anything. The updated upgrade logic has some code to remove the EverestOperator Subscription first and then apply the chart so that the new Deployment can take over..

But if a user has this name hard-coded somewhere in their infrastructure (like CI, scripts, or code that depends on it).. then yes, it could break that.

Maybe we can document this in the upgrade guide. Optionally, we can go back to the original name (which I don't like either BTW 😂 ). Do you have any specific scenario you're particularly concerned about?

Reply:
I'm not concerned about any particular scenario I just don't like `percona-everest` because it's not descriptive. I'd rather have `everest-operator` and `everest-api-server` or `everest-server` :shrug:

Reply:
sorry, I thought we were still talking about the operator name 🤦🏻 

but I don't think my answer changes much.. internally, there shouldn't be breaking anything. During upgrades, we can ensure that we delete the old deployment first, just like we do with the operator. If an existing user depends on the name in someway, it could break things (but we can document it ..)

I'm fine with changing the name. We would have to put in some extra effort while testing though..